src/main/kotlin/org/wycliffeassociates/otter/jvm/persistence/database/daos/ContentDao.kt

```diff
@@ -20,6 +20,17 @@ class ContentDao(
                 }
     }
 
+    fun fetchVerseByCollectionIdAndStart(collectionId: Int, start: Int, dsl: DSLContext = instanceDsl): ContentEntity? {
+        return dsl
+                .select()
+                .from(CONTENT_ENTITY)
+                .where(CONTENT_ENTITY.COLLECTION_FK.eq(collectionId))
+                .and(CONTENT_ENTITY.START.eq(start))
+                .and(CONTENT_ENTITY.LABEL.eq("verse")) // TODO: Probably shouldn't be hardcoded
```

Comment:
I wouldn't go by the label.

For example, OBS has TN as well. Understanding we currently cannot parse OBS to import it anyway, if we did, it has chunks, not verses.

Reply:
Unfortunately I'm not sure how to handle this right now, because we currently are putting a Content entry into the DB that corresponds to a chapter, and which has a start of "0". For now, label is the only differentiating factor (see the image of the Content table below.) That content entry corresponding to the chapter was being placed there in the previous version of the importer, and I'm not sure what purpose it has. Perhaps it shouldn't be placed there in the first place. Is it ok if we create a story to address these issues and leave it the way it is now for the PR? (It's ok if the answer is no.)
![image](https://user-images.githubusercontent.com/11142492/52684167-403af300-2f13-11e9-8143-2dfdce3fa3e1.png)